src/EnergyPlus/ExhaustAirSystemManager.cc

```diff
@@ -451,17 +446,11 @@ namespace ExhaustAirSystemManager {
         }
     }
 
-    void ReportExhaustAirSystem([[maybe_unused]] int &ExhaustAirSystemNum) // maybe unused
-    {
-    }
-
```

Comment:
[src/EnergyPlus/ExhaustAirSystemManager.cc:454]:(style),[unusedFunction],The function 'ReportExhaustAirSystem' is never used.